src/Amazon.Common.DotNetCli.Tools/Utilities.cs

```diff
@@ -195,14 +195,62 @@ public static string LookupTargetFrameworkFromProjectFile(string projectLocation
             return element?.Value;
         }
 
+        /// <summary>
+        /// Retrieve the `OutputType` property of a given project
+        /// </summary>
+        /// <param name="projectLocation">Path of the project</param>
+        /// <returns>The value of the `OutputType` property</returns>
         public static string LookupOutputTypeFromProjectFile(string projectLocation)
         {
-            var projectFile = FindProjectFileInDirectory(projectLocation);
+            var process = new Process();
+            var outputJson = string.Empty;
```

Comment:
nit: As long as we're just getting a single property, it should be the value. I'd drop `Json` from the variable name to avoid confusion.
> If you use -getProperty to request a single property, the output is emitted as a string of text. That means you can easily consume the value in scripting scenarios, or within a more complex command line.

Reply:
Done